pkg/util/paramtable/autoindex_param.go

```diff
@@ -170,7 +170,7 @@ func (p *autoIndexConfig) init(base *BaseTable) {
 	p.ScalarAutoIndexParams = ParamItem{
 		Key:          "scalarAutoIndex.params.build",
 		Version:      "2.4.0",
-		DefaultValue: `{"int": "HYBRID","varchar": "HYBRID","bool": "BITMAP", "float": "INVERTED"}`,
+		DefaultValue: `{numeric": "INVERTED","varchar": "INVERTED","bool": "INVERTED"}`,
```

Comment:
better to keep bool as BITMAP

Reply:
keep all same as 2.4 is better? May we need user to set it by themself which is better for compatibility，they know what changed.